go/client/cmd_paperprovision.go

```diff
@@ -23,16 +23,23 @@ func NewCmdPaperProvision(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cl
 		Action: func(c *cli.Context) {
 			cl.ChooseCommand(NewCmdPaperProvisionRunner(g), "paperprovision", c)
 		},
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "keep-paper-key, k",
```

Comment:
I prefer when a boolean flag says what you want rather than what you don't want, but I don't really care either way.